public/js/controllers.js

```diff
@@ -453,18 +455,20 @@ var updateProjectsPanel = function(result, source){
       }
       // if the project has a migration_id, see if there is a /migrated item
       // with the same id, if so, unlock it
-      if(sourceProject.migration_id){
-        var migrationDone = _.findWhere(sortedMigrated, {migration_id: sourceProject.migration_id});
-        if(migrationDone) {
-          sourceProject.migrating = false;
-          sourceProject.stateSelectionExists =false;
-          sourceProject.selectDestinationType = {};
-          sourceProject.stateHasTools = false;
-          if (sourceProject.tool_id !=='') {
-            var index = $scope.sourceProjects.indexOf(sourceProject);
-            $scope.sourceProjects.splice(index, 1); 
-          }
+      if(sourceProject){
```

Comment:
You probably mean 456 and 458.  If so it is checking for two things, first for a migration_id in the source project, second for a /migrated item with the same migration_id